core/ir/decode_shared.c

```diff
@@ -179,17 +179,17 @@ int sve_veclen;
 int sve_veclens[] = { 128,  256,  384,  512,  640,  768,  896,  1024,
                       1152, 1280, 1408, 1536, 1664, 1792, 1920, 2048 };
 
-void
+bool
 dr_set_sve_vector_length(int vl)
 {
     /* TODO i#3044: Vector length will be read from h/w when running on SVE. */
```

Comment:
This `TODO` is out-of-date. Please delete.

Reply:
Done.